indico/modules/events/registration/util.py

```diff
@@ -476,3 +476,11 @@ def get_event_regforms(event, user, with_registrations=False):
                                                       Registration.user == user,
                                                       Registration.is_active))
     return query.all()
+
+
+def get_ticket_attachment(registration):
```

Comment:
Since this returns a list, wouldn't it be clearer to call this `get_ticket_attachments` (even though it returns only one attachment)?

Otherwise I'd return it without the list around it and wrap it into a list when calling it.